topics/configuration/README.md

```diff
@@ -271,10 +271,10 @@ to wait for one or more nodes to respond back to the coordinator node before a
 request can complete. In multi-datacenter configurations, consistency levels such as
 `EACH_QUORUM` can cause a request to wait for replication across a slower cross
 datacenter network link.  More information about setting the consistency level
-can be found [here](http://datastax.github.io/cpp-driver/topics/basics/consistency/).
+can be found [here](http://docs.datastax.com/en/developer/cpp-driver/latest/topics/basics/consistency/).
 
-[`allow_remote_dcs_for_local_cl`]: http://datastax.github.io/cpp-driver/api/struct.CassCluster/#1a46b9816129aaa5ab61a1363489dccfd0
+[`allow_remote_dcs_for_local_cl`]: http://docs.datastax.com/en/developer/cpp-driver/latest/api/struct.CassCluster/#1a46b9816129aaa5ab61a1363489dccfd0
```

Comment:
Should be `http://docs.datastax.com/en/developer/cpp-driver/latest/api/struct.CassCluster/#function-cass_cluster_set_load_balance_dc_aware` in order to get to the `allow_remote_dcs_for_local_cl` parameter description